src/SFA.DAS.ReleaseTool.Web/Views/Home/Index.cshtml

```diff
@@ -0,0 +1,36 @@
+@{
+    ViewData["Title"] = "Home";
+    @model IndexViewModel
+    @using SFA.DAS.ReleaseTool.Web.Controllers
+    @using SFA.DAS.ReleaseTool.Web.Extensions
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-xl">
+            DAS Whitelist
+        </h1>
+
+        <form asp-controller="@typeof(HomeController).GetControllerName()" asp-action="@nameof(HomeController.StartRelease)" method="post">
```

Comment:
@chelnak Did we decide we'd rather use RouteName constants instead of this?

Reply:
Yeah preferably. Tools is transitioning.